packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -538,6 +541,8 @@ class ThemeData with Diagnosticable {
     timePickerTheme ??= const TimePickerThemeData();
     toggleButtonsTheme ??= const ToggleButtonsThemeData();
     tooltipTheme ??= const TooltipThemeData();
+    // DEPRECATED (newest deprecations at the bottom)
+    buttonBarTheme ??= const ButtonBarThemeData();
```

Comment:
Ah! Just when ThemeData was clear! 😆

Reply:
Haha yes, i saw your comment [here](https://github.com/flutter/flutter/pull/144078/files#r1515110959) when I had to restore all the deprecation sections with comments.